cli/default-reporter/src/reporterForClient/reportSummary.ts

```diff
@@ -71,6 +71,11 @@ export function reportSummary (
             msg += EOL
             msg += _printDiffs(diffs)
             msg += EOL
+            if (depType === 'dev' && opts.pnpmConfig?.saveDev === false && opts.pnpmConfig?.extraEnv?.npm_command === 'add') {
+              msg += EOL
+              msg += `The dependency was already listed in devDependencies.${EOL}If you want to make it a prod dependency, then move it manually.`
```

Comment:
Is there any chance to show the dep directly? E.g. Adding 2 pkg same time while only showing the existing one?

Reply:
here the result :  (may be we can change the message with ` was not moved, already listed in devDependencies`

Reply:
Here with a better message imho :

![image](https://github.com/pnpm/pnpm/assets/106020312/f6dda777-cf87-423d-9f66-5ba20e2b2b66)

Reply:
I'm not sure on the messaging, its not clear (at least in my own opinion) - the first message gave us some sort of idea that this was a duplicate dependency - this new one makes it read as if pnpm is moving packages without us being aware

Reply:
I understand, i can update the message

@zkochan a recommendation regarding the message ?